src/Root.tsx

```diff
@@ -25,7 +25,7 @@ const Content = () => {
       i18n={i18n}
       langs={langs}
       languageSelectSx={{ mb: 2, mr: 2 }}
-      languageSelectLabel={null}
+      languageSelectLabel=""
```

Comment:
It is possible to provide nothing? Otherwise, it's okay like that

```suggestion
```

Reply:
It would be possible but currently, if nothing is provided, a default value is used and it is not translated.